samples/application-conf/README.md

```diff
@@ -23,6 +23,11 @@ gitRepositoryURL | git repository URL of the application repository to establis
 excludeFileList | Files to exclude when scanning or running full build. | false
 skipImpactCalculationList | Files for which the impact analysis should be skipped in impact build | false
 jobCard | JOBCARD for JCL execs | false
+**Build Property management** | | 
+loadFileLevelProperties | Flag to enable the zAppBuild capability to load individual property files for a build file | true
+propertyFilePath | relative path to folder containing individual property files | true
+propertyFileExtension | file extension for individual property files | true
```

Comment:
- How can the new Build Property management properties be overridden?

- We might also want to update the **file.properties** section below where it currently says:
  > All file properties for the entire application, including source files in distributed repositories of the application need to be contained either in this file or in other property files in the `application-conf` directory.
  
  Should it be updated to additionally mention the directory specified by the `propertyFilePath` property of `application-conf/application.properties`?